### PR TITLE
'Dogecoin FAH Article'

### PR DESCRIPTION
**Pull request by RichieJohn**
Note that I haven't double checked this with the current project maintainers.

### CODE REVIEW

**Review by xanimo — APPROVED** [review]
ACK

File: content/dogepedia/articles/dogecoin-folding-at-home.md [hunk 1]
```diff
@@ -5,4 +5,8 @@ date = "2021-10-22"
   name = "Dogecoin"
 +++
 
-Describe the Dogecoin Folding @ Home - but before including this, double check with the current project maintainers. This article could be contributed by them.
\ No newline at end of file
+Dogecoin Folding at Home is a community-based distributed-computing charitable project that utilises participants’ idle computer power to help scientists who are studying Covid, Alzheimer's, Huntington's, Parkinson's, cancers, and more., as they work towards finding cures.
```
Comment (xanimo) [inline content/dogepedia/articles/dogecoin-folding-at-home.md:8]: cancers and more as they work towards finding cures.*

Comment (xanimo) [inline content/dogepedia/articles/dogecoin-folding-at-home.md:8]: 'charitable community-based distributed-computing project'?

File: content/dogepedia/articles/dogecoin-folding-at-home.md [hunk 2]
```diff
@@ -5,4 +5,8 @@ date = "2021-10-22"
...
\ No newline at end of file
+Dogecoin Folding at Home is a community-based distributed-computing charitable project that utilises participants’ idle computer power to help scientists who are studying Covid, Alzheimer's, Huntington's, Parkinson's, cancers, and more., as they work towards finding cures.  
+
+It uses a very simple GUI interface, and runs in the background using idle computer time via GPUs, CPUs, and ARM processors. Your computer will receive “work units”, and then sends the completed work units back to the FAH servers. These are then compiled with other work units into an overall statistical simulation pertaining to one or more diseases and their potential therapies.   
+
+In addition to helping scientists develop new therapeutics for a variety of diseases, participants are also rewarded with weekly payouts in Dogecoin, which are provided to the organisation by long term sponsors and user donations. Dogecoin Folding at Home supports Windows, Linux, and MacOS operating systems, and to get started or learn more: [click here] (https://www.dogecoinfah.com/index.html)!
```
Comment (xanimo) [inline content/dogepedia/articles/dogecoin-folding-at-home.md:12]: '...MacOS operating systems. To get started or learn more:...'?

File: content/dogepedia/articles/dogecoin-folding-at-home.md [hunk 3]
```diff
@@ -5,8 +5,8 @@ date = "2021-10-22"
...
 
-In addition to helping scientists develop new therapeutics for a variety of diseases, participants are also rewarded with weekly payouts in Dogecoin, which are provided to the organisation by long term sponsors and user donations. Dogecoin Folding at Home supports Windows, Linux, and MacOS operating systems, and to get started or learn more: [click here] (https://www.dogecoinfah.com/index.html)! 
\ No newline at end of file
+In addition to helping scientists develop new therapeutics for a variety of diseases, participants are also rewarded with weekly payouts in Dogecoin, which are provided to the organisation by long term sponsors and user donations. Dogecoin Folding at Home supports Windows, Linux, and MacOS operating systems. To get started or learn more: [click here] (https://www.dogecoinfah.com/index.html)!
```
Comment (xanimo) [inline content/dogepedia/articles/dogecoin-folding-at-home.md:12]: just need to remove the space in between `[click here] (https://www.dogecoinfah.com/index.html)` and it looks good to go!

Comment (RichieJohn) [reply content/dogepedia/articles/dogecoin-folding-at-home.md:12]: What space do you mean @xanimo? Ready to change now :)

**Pull request merged**